relentless/optimize/method.py

```diff
@@ -280,57 +287,157 @@ def max_iter(self, value):
 class SteepestDescent(Optimizer):
     r"""Steepest descent algorithm.
 
-    For an :class:`~relentless.optimize.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
     the steepest descent algorithm seeks to approach a minimum of the function.
 
-    With an initial numerical guess for all the design variables :math:`\mathbf{x}`,
-    the following iterative calculation is performed:
+    :math:`\mathbf{X}=\left[X_1,\ldots,X_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`y_i=\frac{x_i}{X_i}`. By
+    default, the variables are left unscaled (:math:`X_i=1`).
+
+    With scaling, the gradient of the function becomes:
 
     .. math::
 
-        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{x}\right)
+        \nabla f\left(\mathbf{y}\right) = \left[X_1 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                X_n \frac{\partial f}{\partial x_n}\right]
 
-    where :math:`\alpha` is defined as the step size hyperparameter. The step size
-    must be specified numerically, and a :class:`LineSearch` can be performed to find
+    :math:`\alpha` is defined as the step size hyperparameter. The step size must
+    be specified numerically, and a :class:`LineSearch` can be performed to find
     an optimal step size value if desired.
 
+    Then, with an initial numerical guess for all the design variables :math:`\mathbf{x}`,
+    the following iterative calculation is performed:
+
+    .. math::
+
+        \mathbf{y}_{n+1} = \mathbf{y}_{n}-\alpha\nabla f\left(\mathbf{y}\right)
```

Comment:
This is out of order, it should come after line 295 so that you define the update scheme. I also don't think it's necessary to include the second line up front.
```suggestion
    The function is iteratively minimized by taking successive steps down
    the gradient of the functional. If the scaled variables are :math:`\mathbf{y}_n`
    at iteration *n*, the next value of the variables is:

    .. math::

        \mathbf{y}_{n+1} = \mathbf{y}_{n}-\alpha\nabla f\left(\mathbf{y}\right)
```
Then put the part from lines 299-305.